oteapi/strategies/filter/crop_filter.py

```diff
@@ -45,12 +45,18 @@ class CropImageFilter:
 
     filter_config: CropImageFilterConfig
 
-    def initialize(self, session: "Optional[Dict[str, Any]]" = None) -> SessionUpdate:
+    def initialize(
+        self,
+        session: "Optional[Dict[str, Any]]" = None,
+    ) -> SessionUpdateCropFilter:
         """Initialize strategy and return a dictionary."""
-        return SessionUpdate()
+        return SessionUpdateCropFilter(
+            imagecrop=self.filter_config.configuration.crop,
+        )
```

Comment:
We've agreed offline that `initialize` is the method to be used for filters (and mappings), since this allows them to be used in the most flexible way (both before or after a "source" strategy).